src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -2895,9 +2900,19 @@ export function getChartSettingsMap(
         }
         chartSettingsMap[id] = chartSetting;
     });
+    // attributes disabled on the summary tab (used in Clinical Data tab but not Summary tab like most survival attributes)
+    const disabledAttributes = _.differenceWith(
+        visibleAttributes,
+        visibleAttributesForSummary,
+        _.isEqual
+    );
     // add layout for each chart
     gridLayout.forEach(layout => {
-        if (layout.i && chartSettingsMap[layout.i]) {
+        if (
+            layout.i &&
+            chartSettingsMap[layout.i] &&
```

Comment:
would it be possible to avoid putting these disabled attributes into chartSettingsMap in the first place so we wouldn't have to check this here?

Reply:
we still want to keep the disabled attributes in the map for user settings purposes for `Clinical Data` tab columns, just without the layout or else it will mess with chart layout on summary tab.